api/src/db/seeder.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose')
+const Account = require('./Account')
+const Wish = require('./Wish')
+const chance = require('chance').Chance()
+const wishRouter = require('../routes/wish')
+var _ = require('lodash');
+
+
+process.env.mongoUrl = process.env.mongoUrl || 'mongodb://localhost:27017/my_database'
+require('./bootstrap-mongoose')
+
+createAccounts = async () => {
+
+  await Account.find({}).remove(async function() {
+
+    const userAdmin = new Account({username: 'Admin', role: 'admin'});
+    await userAdmin.setPassword('Password321');
+    await userAdmin.save();
+    await Account.authenticate()('Admin', 'Password321');
+
+    let i = 0
```

Comment:
This can be simplified a bit without needing to declare the iterator and the count separately.

[...Array(10)].forEach((_, i) => {
  // do something here 10x
}